src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -94,7 +96,8 @@ export default function LineChartFactory(Private) {
           const height = self.baseChart.chartConfig.height;
           const circleRadius = (d.z - radii.min) / radiusStep;
 
-          return _.min([Math.sqrt((circleRadius || 2) + 2), width, height]) + (modifier || 0);
+          const base = circleRadius ? Math.sqrt(circleRadius + 2) + lineWidth : lineWidth;
```

Comment:
Right, that makes sense - thanks.

The number seems to be an offset that makes sure the circle diameter is always larger than the line width. Not sure why it's inside the `sqrt` though. I would call the constant `CIRCLE_RADIUS_OFFSET` or something like that.